st2common/bin/st2ctl

```diff
@@ -141,6 +141,7 @@ function clean_logs() {
 function getpids() {
   echo "##### st2 components status #####"
   COMPONENTS=${COMPONENTS/mistral/mistral-server mistral-api}
+  COMPONENTS=${COMPONENTS/st2chatops/hubot}
```

Comment:
when it exists, it picks up two pids: one from the parent shell wrapper, another from actual node. Not sure if we need two pids, it's a bit confusing.
```
ps ax | grep hubot
31774 ?        S      0:00 bash -c ulimit -S -c 0 >/dev/null 2>&1 ; cd /opt/stackstorm/chatops && nohup </dev/null >/dev/null 2>&1 /opt/stackstorm/chatops/bin/hubot &
31776 ?        Sl     0:01 node /opt/stackstorm/chatops/node_modules/hubot/node_modules/.bin/coffee /opt/stackstorm/chatops/node_modules/.bin/hubot @
```